mosdef_cassandra/writers/inp_functions.py

```diff
@@ -1899,3 +1922,46 @@ def _check_restricted_insertions(box, restriction_type, restriction_value):
                 " for 'interface' is"
                 " greater than the z-coordinate of the box."
             )
+
+
+def _convert_kwarg_units(kwargs):
+    """Convert kwargs that are unyt units
+    """
+    if "vdw_cutoff" in kwargs:
+        kwargs["vdw_cutoff"] = kwargs["vdw_cutoff"].to("angstrom")
+    if "vdw_cutoff_box1" in kwargs:
+        kwargs["vdw_cutoff_box1"] = kwargs["vdw_cutoff_box2"].to("angstrom")
+    if "vdw_cutoff_box2" in kwargs:
+        kwargs["vdw_cutoff_box2"] = kwargs["vdw_cutoff_box2"].to("angstrom")
+    if "charge_cutoff" in kwargs:
+        kwargs["charge_cutoff"] = kwargs["charge_cutoff"].to("angstrom")
+    if "rcut_min" in kwargs:
+        kwargs["rcut_min"] = kwargs["rcut_min"].to("angstrom")
+    if "pressure" in kwargs:
+        kwargs["pressure"] = kwargs["pressure"].to("bar")
+    if "pressure_box1" in kwargs:
+        kwargs["pressure_box1"] = kwargs["pressure_box1"].to("bar")
+    if "pressure_box2" in kwargs:
+        kwargs["pressure_box2"] = kwargs["pressure_box2"].to("bar")
+    if "chemical_potentials" in kwargs:
+        new_mu = list()
+        for mu in kwargs["chemical_potentials"]:
+            mu = mu.to("kJ/mol")
+            new_mu.append(mu)
+        kwargs["chemical_potentials"] = new_mu
+    if "cbmc_rcut" in kwargs:
+        kwargs["cbmc_rcut"] = kwargs["cbmc_rcut"].to("angstrom")
+
+    return kwargs
+
+
+def _convert_moves_units(moves):
+    new_max_volume = list()
+    for max_vol in moves.max_volume:
+        max_vol = max_vol.to("angstrom**3")
+        new_max_volume.append(max_vol)
+    moves.max_volume = new_max_volume
+
+    # Add conversion for restricted_value
+
+    return moves
```

Comment:
Should these functions go in their own file? If someone wants to see the unit conversions we are doing I think it would be beneficial for them to be easy to find.

Reply:
I could move these to `utils/unit.py`, which might be easier to find.